.changeset/witty-waves-rhyme.md

```diff
@@ -0,0 +1,5 @@
+---
+'astro': minor
+---
+
+Removes image service override, in favor to moving that logic into adapters and fix the regression it introduced
```

Comment:
```suggestion
Moves the logic for overriding the image service out of core and into adapters.  Also fixes a regression where  a valid `astro:assets` image service configuration could be overridden.
```
Maybe something like this? (I know the other adapter 33 PR has not finalized its changeset wording yet. So, whatever you decide there might affect what you put here.)